hal/targets.json

```diff
@@ -1823,7 +1823,7 @@
         "inherits": ["Target"],
         "core": "Cortex-M0+",
         "default_toolchain": "uARM",
-        "supported_toolchains": ["GCC_ARM", "uARM"],
+        "supported_toolchains": ["GCC_ARM", "uARM", "IAR"],
```

Comment:
Hmm I wouldn't expect that to be the case for the ARM toolchain, I would for GCC though.

Is this still true? I believe a few memory optimization PRs have been merged relatively recently.